src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const minDiscont=3;
+  const baseDiscont=7;
+  const baseCost=40*days;
+  if (days<minDiscont){
+    return baseCost;
+  }
+  if (days>=baseDiscont){
+    return baseCost-50;
```

Comment:
Consider defining `50` as a constant, such as `BASE_DISCOUNT_AMOUNT`, to improve readability and maintainability.